nss-cohort-36/keahua-arboretum-net-surfers
# Ts tuesday menu fun

# Description

Add planet menu restrictions.

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout TS-Tuesday-Menu-Fun`
4. Create 2 mountain biomes, add mountain plant 4 times to one biome, check and see if you are prevented from adding plant to full biome, 

# Checklist:
